### PR TITLE
Fix menu builder

### PR DESCRIPTION
**Pull request by craigharman**
use TCG\Voyager\Voyager; was missing from menu builder controller.

### CODE REVIEW

File: src/Http/Controllers/VoyagerMenuController.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 use Illuminate\Http\Request;
 use TCG\Voyager\Models\Menu;
 use TCG\Voyager\Models\MenuItem;
+use TCG\Voyager\Voyager;
```
Comment (marktopper) [inline src/Http/Controllers/VoyagerMenuController.php:8]: I don't see this in use in `VoyagerMenuController`.

**Comment by craigharman** [comment]
I get an exception when trying to edit the menu without it. Line 28. 

Kind regards,

Craig




On Mon, Dec 5, 2016 at 9:26 PM +0800, "Mark Topper" <notifications@github.com> wrote:












@marktopper commented on this pull request.





In src/Http/Controllers/VoyagerMenuController.php:
> @@ -5,6 +5,7 @@
 use Illuminate\Http\Request;
 use TCG\Voyager\Models\Menu;
 use TCG\Voyager\Models\MenuItem;
+use TCG\Voyager\Voyager;



I don't see this in use in VoyagerMenuController.

**Comment by marktopper** [comment]
I see, it just showed me the base from the `master` branch instead of the `v0.10` branch.

However can you fix the conflicts or make a new pull request for this, then I will merge this in.

**Comment by fletch3555** [comment]
@marktopper, I think I may have fixed this in #309

**Comment by marktopper** [comment]
Because of #308

**Pull request closed without merging**